source/fundamentals/connection/connect.txt

```diff
@@ -4,8 +4,9 @@
 Connect to MongoDB
 ==================
 
-In this guide, you can learn how to connect to a MongoDB instance or
-replica set using the Java driver.
+This guide shows you how to connect to a 
```

Comment:
Nitpick/Suggestion: I think the phrase "This guide shows you" is personifying the page. It seems more clear to me to stick with the "you can learn" format. (This is probably pretty subjective so feel free to take or leave this comment)

```suggestion
In this guide, you can learn how to connect to a 
```